express/src/middleware/validators/emailValidator.ts

```diff
@@ -0,0 +1,18 @@
+import {NextFunction, Request, Response} from "express";
+import {validateEmail} from "../../lib/validators/emailValidator";
+import {validationResult} from "../../lib/validators/validationResult";
+type MiddlewareFunction<T, U, V> = (T: Request, U: Response, V: NextFunction) => void;
+
+export function emailValidator(template: string): MiddlewareFunction<Request, Response, NextFunction> {
```

Comment:
```suggestion
export function emailValidator(template: string): MiddlewareFunction<Request, Response, NextFunction> {
```

I am still not sure about passing templates into the middlewares (something to think about later)